internal/build/docker_test.go

```diff
@@ -0,0 +1,154 @@
+package build
+
+import (
+	"context"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+
+	"github.com/windmilleng/wat/os/temp"
```

Comment:
???

this seems wrong

Reply:
Yeah this was a bit of a hack to get all that functionality. Do you think we should copy it in or move it to wmclient @nicks?